<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnsEditDialog</class>
 <widget class="QDialog" name="ColumnsEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Колонки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QLabel" name="dialog_title">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Настройка колонок</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttons_layout">
       <property name="spacing">
        <number>100</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ok_button">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
